docs/cli.md

```diff
@@ -64,54 +68,60 @@ The general structure for running commands with the Gravitino CLI is `gcli entit
  -x,--index              display index information
  -z,--provider <arg>     provider one of hadoop, hive, mysql, postgres,
                          iceberg, kafka
- ```
+```
 
-## Commands
+## Commands Types
```

Comment:
I would not use "Types" to describe commands. Perhaps "Command Actions"?

Reply:
Revised.